### PR TITLE
Assignments resolution

### PR DESCRIPTION
**Pull request by martinzugnoni**
- Many UI tweaks to make it pretty.
- Assignment resolution using external execution code service.

### CODE REVIEW

File: rmotr_sis/courses/forms.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from __future__ import division, unicode_literals, absolute_import
```
Comment (santiagobasulto) [inline rmotr_sis/courses/forms.py]: Aren't we using Python-3.4?

File: rmotr_sis/courses/views.py [hunk 2]
```diff
@@ -44,3 +52,81 @@ def validate_user_in_courseinstance(self):
...
+    def get_initial(self):
+        self.assignment = get_object_or_404(Assignment, pk=self.kwargs['pk'])
+        return {'source': self.assignment.source}
+
+    def get_context_data(self, **kwargs):
+        context = super(ResolveAssignmentView, self).get_context_data(**kwargs)
+
+        # try to find an unfinished attempt for this assignment. If nothing
+        # is found, create a new one.
+        obj, created = AssignmentAttempt.objects.get_or_create(
```
Comment (santiagobasulto) [inline rmotr_sis/courses/views.py]: Beautiful

**Comment by santiagobasulto** [comment]
+1

**Comment by AmaxJ** [comment]
+1000

**Pull request merged**